<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtEzanvakti</class>
 <widget class="QMainWindow" name="QtEzanvakti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtEzanvakti</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>551</width>
      <height>281</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_v">
     <attribute name="title">
      <string>Vakitler</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>181</width>
        <height>201</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_s">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sabah</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_a">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>58</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Akşam</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_i">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>58</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>İkindi</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_y">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>58</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Yatsı</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_o">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Öğle</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_g">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Güneş</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_sv">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_yv">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_iv">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_av">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ov">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_gv">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_mv">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>16</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_k1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kerahat Vakti 1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_k3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kerahat Vakti 3:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_k4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kerahat Vakti 4:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_k5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kerahat Vakti 5:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_k2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kerahat Vakti 2:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_kv1">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_kv2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>100</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_kv5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>160</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_kv3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>120</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_kv4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>140</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_g">
     <attribute name="title">
      <string>Günlük</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_d">
     <attribute name="title">
      <string>Dinle</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_ul">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_il">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_st">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ss">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ezv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
